render-configs.sh

```diff
@@ -1,32 +1,161 @@
 #!/bin/bash
+### How to call this script
```

Comment:
TOL (not required change): At some point we may want to think about converting this to python - I think it might give us better access to help text and a bit more help for function composition.